### DISCUSSION TITLE
Video element

### DISCUSSION THREAD

**Discussion by chris77erickson** [webstudio-is/webstudio-community#27] [Wishlist]
And even better, the video element is a lightbox element by default.

**Comment by kof** [comment]
Yep, definitely need one. We need to decide which services to support out of the box, my bet is youtube and vimeo, but we probably also need custom video upload for short videos. Youtube is google and privacy is our priority.

**Reply by Erudition** [reply]
Archive.org is a FLO option for 3rd party hosting.
Peertube is a great FLO option for skipping most of the hosting burden altogether.

**Comment by kof** [comment]
Follow this issue https://github.com/webstudio-is/webstudio-builder/issues/1688 for implementation details

**Comment by kof** [comment]
Btw first Video element with Vimeo is in production, check it out

**Reply by kwameandco** [reply]
Any chance of something like a proper video element?
e.g.
<video width="320" height="240" autoplay muted>
  <source src="movie.mp4" type="video/mp4">
  <source src="movie.ogg" type="video/ogg">
Your browser does not support the video tag.
</video>

Where the properties are baked into the designer (e .g. toggles etc for autoplay/muted).
Also a "choose file" option for the src if it's uploaded directly, like for the img element?
Yes, I know I can just use an embed / "element" and assign the video tag, but again, this is Quality of Life.
